api/source/output_formats/formats.rst

```diff
@@ -111,17 +80,21 @@ The following defines the Job in JSON format:
     {
         "JobId" : "<Guid HexString>",
         "JobName" : "<String>",
-        "MediaLengthSeconds" : "<Integer>",
+        "Username": "<String>",
+        "MediaLengthSeconds" : "<Float>",
         "ExternalID" : "<String (dependant on third-party integrations)",
-        "JobStatus" : "<Status Enum>",
         "Priority" : "<Priority Enum>",
         "Fidelity" : "<Fidelity Enum>",
-        "JobLanguage" : "<RFC 5646 Language Code>",
+        "JobStatus" : "<Status Enum>",
```

Comment:
Can we change this to "<Job Status Enum>" to keep it consistent with the enum name?